src/style.css

```diff
@@ -1 +1,8 @@
 /* styles go here */
+.form ~ .form {
+  margin-top: 10px;
+}
+
+fieldset ~ fieldset {
+  margin-top: 20px;
```

Comment:
use just classes

Reply:
why do you need `~`?

Reply:
Here the task was to make a distance between inputs and fieldsets , so I applied Styles only for those elements that go after first one, so I don't have margin-top on the first one